### ISSUE TITLE
better availability check

### ISSUE THREAD

**Issue by tyrollins** [zoetrope69/domains#31]
neat app, been looking for something like this for a while. currently though most of the domains it displays are registered... I found 1 available, the rest are all taken.

**Comment by zaccolley** [comment]
are you running this locally?

**Comment by tyrollins** [comment]
Yes running locally. It seems to have been a problem with connecting to the dictionary api as it's not in mashapes marketplace last I checked.

After poking about a bit more, I realized I had the concept wrong...I was thinking more of a hipster domain finder type deal where the app presented primarily available domains. Not to worry though, part of the fun is in the search! Again great app!

**Comment by zoetrope69** [comment]
I'll keep this open because i think that'd be a cool feature but reality is probably wont get round to doing it anytime soon

**Comment by zoetrope69** [comment]
i've got this live over here https://domains.glitch.me/ too now if that makes it easier for you
